ep/build-velox/src/get_velox.sh

```diff
@@ -16,8 +16,8 @@
 
 set -exu
 
-VELOX_REPO=https://github.com/oap-project/velox.git
-VELOX_BRANCH=2025_09_02
+VELOX_REPO=https://github.com/JkSelf/velox.git
+VELOX_BRANCH=smj-test.0903
```

Comment:
Need to resolve the conflicts after pick velox pr https://github.com/facebookincubator/velox/pull/14685